doc/developer-guides/trusty.rst

```diff
@@ -0,0 +1,186 @@
+    :        |        |                                                :        :  +-------+                  |        |         :
+    :        |        |                                                :        :  |       |                  |        |         :
+    :        | 0-512G |--+                                             :        :  |       |               +--| 0-512G |         :
+    :EPTP -->+--------+  |                                             :        :  |       |               |  +--------+<-- EPTP :
+    :                    |      PDPT                                   :        :  |       |      PDPT     |                     :
+    :                    |   +--------+                                :        :  |       |   +--------+  |                     :
+    :                    |   | >=511G |---> Not present                :        :  +-------+<--| >=511G |  |                     :
+    :                    |   |________|                                :        :              |________|  |                     :
+    :                    |   |        |                                :        :              |        |  |                     :
+    :                    |   | <511G  |->+<----------------------------:--------:--------------| <511G  |  |                     :
+    :                    |   |        |  |                             :        :              |        |  |                     :
+    :                    +-->+--------+  |        PD              PT   :        :              +--------+<-+                     :
+    :                                    |       ...             ...   :        ==================================================
+    :                                    |    +-------+      +-------+ :
+    :                                    |   +-------+|     +-------+| :
+    :                                    |   |       ||     |       || :
+    :                                    |   |       ||     |       || :
+    :                                    |   |  PDE  |--+   |       || :
+    :                                    |   |       || |   |       || :
+    :                                    |   |       |+ |   |       |+ :
+    :                                    +-->+-------+  +-->+-------+  :
+    :                                                                  :
+    ====================================================================
+
+
+API
+===
+
+.. code-block:: c
+
+   /*
+   Create Secure World EPT hierarchy, construct new PML4/PDPT, reuse PD/PT parse from
+   vm->arch_vm->ept
+
+   Parameters:
+       vm:		VM with 2 Worlds
+       gpa:		original gpa allocated from vSBL
+       size:		LK size(16M by default)
+       rebase_offset:	rebase the gpa to offset xxx(511G_OFFSET)
+   */
+
+   int create_secure_world_ept(vm_t *vm, uint64_t gpa, uint64_t size, uint64_t rebase_offset)
```

Comment:
I can't find a ``create_secure_world_ept`` function in any file currently in the hypervisor source tree.  This should also be a reference to doxygen-generated API material and not just written here in the documentation.